docs/CHANGELOG.md

```diff
@@ -13,13 +13,15 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-------------------------------------------------->
 
 
-## PLACEHOLDER FOR CELL TYPING RELEASE DATE
+## PLACEHOLDER FOR CELL TYPING/COMMUNITY CONTRIBUTIONS RELEASE DATE
 
 * Cell type annotations are now included in each download.
 Cells were annotated using both [`SingleR`](https://bioconductor.org/packages/release/bioc/html/SingleR.html) and [`CellAssign`](https://github.com/Irrationone/cellassign).
   * You can find more information about how cell types were annotated in the {ref}`cell type annotation procedures section on the Processing Information page<processing_information:cell type annotation>`.
   For more information on locating cell type annotations and any associated processing information in the downloaded objects see {ref}`the Single-cell gene expression file contents page<processing_information:components of a singlecellexperiment object>`.
 * Downloads will also contain a separate cell type report providing more information about cell type annotations, including comparisons between different cell type annotations and diagnostic assessments of cell type annotation reliability.
+* This release additionally includes community-contributed datasets.
+Like all other projects in the ScPCA Portal, all community-contributed projects are 10x Genomics single-cell or single-nuclei datasets that have been processed with the ScPCA pipeline.
```

Comment:
Maybe we should note that datasets that are community-contributed will be indicated with a badge on the project card?

Reply:
This sounds like a good idea!